recipes/rauth/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: rauth
+  version: "0.7.3"
+
+source:
+  fn: rauth-0.7.3.tar.gz
+  url: https://pypi.python.org/packages/43/aa/7c8e852275394d65ac5bf3ac9945ecaafe4d083089e09cb0a267efea389a/rauth-0.7.3.tar.gz
+  sha256: 524cdbc1c28560eacfc9a9d40c59525eb8d00fdf07fbad86107ea24411477b0a
+  
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - requests >=1.2.3
+
+  run:
+    - python
+    - requests >=1.2.3
+
+test:
+  imports:
+    - rauth
+
+about:
+  home: https://github.com/litl/rauth
+  license: MIT
+  summary: 'A Python library for OAuth 1.0/a, 2.0, and Ofly.'
+  license_family: MIT
```

Comment:
You need to add a list of maintainers including yourself:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L70-L75